src/backend/distributed/commands/table.c

```diff
@@ -1939,12 +1968,19 @@ PostprocessAlterTableStmt(AlterTableStmt *alterTableStatement)
 		 * since this is the only ALTER command of a sequence that
 		 * passes through an AlterTableStmt
 		 */
-		if (get_rel_relkind(relationId) == RELKIND_SEQUENCE)
+		char relKind = get_rel_relkind(relationId);
+		if (relKind == RELKIND_SEQUENCE)
 		{
 			AlterTableStmtObjType_compat(alterTableStatement) = OBJECT_SEQUENCE;
 			PostprocessAlterSequenceOwnerStmt((Node *) alterTableStatement, NULL);
 			return;
 		}
+		else if (relKind == RELKIND_VIEW)
+		{
+			AlterTableStmtObjType_compat(alterTableStatement) = OBJECT_VIEW;
```

Comment:
why do we `copyObject` in other ops but not here? Maybe we should NOT copy at all, because we send workers as `ALTER VIEW` and it is better to run the same on the coordinator?
Not sure, asking your opinion as well

Reply:
We copy it from preprocess functions and use the statement directly from postprocess functions. I found it more safe to copy it within preprocess but don't need to copy it from postpocess. So, I'm keeping it as it is.